sql/03.delete.sql

```diff
@@ -0,0 +1,2 @@
+DELETE FROM tbl_persons 
+    WHERE address = 'unknown';
```

Comment:
address 'unknown' is bad convention for absent addresses when you have null value